api/settings/base.py

```diff
@@ -37,8 +37,11 @@
     'django_nose',
 
     'djangotoolbox',
+    
     'authentication',
     'ext_news',
+    'utils',
```

Comment:
do we need utils as app? we can use it as usual utils

Reply:
deleted for now